ersilia/core/tracking.py

```diff
@@ -348,17 +372,17 @@ def get_peak_memory(self):
         tracemalloc.stop()
 
         return peak_memory
-
+    
     def track(self, input, result, meta):
```

Comment:
> ```python
> def track(self, input_file, result_file, meta):
> ```

I was also able to fix all the changes you recommended except for this because changing the variables was causing an [error](https://github.com/user-attachments/files/15570150/output3.log)
 in the model file and I didn't want to interfere with that since it's beyond the scope of this issue.